tests/performance/test_performance.py

```diff
@@ -0,0 +1,17 @@
+def test_performance():
```

Comment:
I think we can remove this file and the example.json below, since the current PR is not intended to cover test cases yet